aten/src/ATen/native/native_functions.yaml

```diff
@@ -1922,6 +1922,9 @@
     CPU: s_addmm_sparse_dense_cpu_
     CUDA: s_addmm_sparse_dense_cuda_
 
+- func: _sparse_addmm(Tensor self, Tensor sparse, Tensor dense, *, Scalar beta=1, Scalar alpha=1) -> Tensor
```

Comment:
for some reasons torch can't find `_sparse_addmm`, although this PR (https://github.com/pytorch/pytorch/pull/12430/files) works fine. Am I missing something here? @ezyang

Reply:
Err, but CI seems to be passing? How do you reproduce the problem.

Reply:
Thanks! Let me add a test for it. The error "AttributeError: module 'torch' has no attribute '_sparse_addmm'" should show up in the CIs.

Reply:
here's the error: https://circleci.com/gh/pytorch/pytorch/230346?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link